subgraph/src/public-lock.ts

```diff
@@ -83,7 +102,17 @@ function newKey(event: TransferEvent): void {
   }
 
   // create receipt
-  createReceipt(event)
+  createReceipt({
```

Comment:
You should just pass the `event` object and extract what you need from it in the `createReceipt` function.